.travis.yml

```diff
@@ -19,6 +19,8 @@ cache: pip
 before_install:
   - python --version
   - if [[ -n $CC ]]; then $CC --version; fi
+  - sudo apt-get update
+  - sudo apt-get install -y libprotobuf-dev
 install:
   - pip install "poetry<2,>=1.0" tox
   - pip install "grpcio-tools~=1.8" "mypy-protobuf~=1.0"
```

Comment:
We now require protoc because we are generating C++ code. Eliminate references to grpcio-tools since it only supports Python.